ocs_ci/ocs/resources/pvc.py

```diff
@@ -162,6 +162,25 @@ def resize_pvc(self, new_size, verify=False):
             )
         return True
 
+    def get_attached_pods(self):
+        """
+        Get the pods attached to the PVC represented by this object instance
+
+        Returns:
+            list: A list of pod objects attached to the PVC
+
+        Note:
+             importing from POD inside, because of unsolvable import loop
```

Comment:
Add as a comment just before the import statement.
```suggestion
             # Importing from pod inside, because of unsolvable import loop
             from ocs_ci.ocs.resources.pod import get_all_pods, get_pvc_name
```